### PR TITLE
Feature/build matrix py3

### PR DESCRIPTION
**Pull request by the-it**
Hello Oefenweb.nl team,

a little improvement for your ansible test suite, maybe you like it.

### CODE REVIEW

**Comment by tersmitten** [comment]
Looks like a nice improvement

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
-sudo: required
-dist: trusty
+dist: bionic
```
Comment (tersmitten) [inline .travis.yml]: Can you change this to `xenial`?

File: .travis.yml [hunk 2]
```diff
@@ -39,43 +40,6 @@ env:
...
-  - ANSIBLE_VERSION=2.5.9
-  - ANSIBLE_VERSION=2.5.8
-  - ANSIBLE_VERSION=2.5.7
-  - ANSIBLE_VERSION=2.5.6
-  - ANSIBLE_VERSION=2.5.5
-  - ANSIBLE_VERSION=2.5.4
-  - ANSIBLE_VERSION=2.5.3
-  - ANSIBLE_VERSION=2.5.2
-  - ANSIBLE_VERSION=2.5.1
-  - ANSIBLE_VERSION=2.5.0
```
Comment (tersmitten) [inline .travis.yml:78]: Any reason these are removed?

File: .travis.yml (hunk 1 above)
Comment (the-it) [reply .travis.yml]: that's possible

File: .travis.yml (hunk 2 above)
Comment (the-it) [reply .travis.yml:78]: This versions are no longer maintained by Ansible. The further check for Python 3 was doubling the builds. So I thought, don't stress Travis to much.

**Comment by the-it** [comment]
It's also necessary to exclude the version 2.7 from the tests with Python3. These packages are not PY3 compliant. Is it really necessary to check every bugfix level of Ansible?

**Comment by the-it** [comment]
Want to have this PR for your other repos as well?

**Comment by the-it** [comment]
@tersmitten, is there any interest in this change?

**Comment by tersmitten** [comment]
Python 3 is added recently

**Pull request closed without merging**